types/koa-route/package.json

```diff
@@ -0,0 +1,5 @@
+{
+    "dependencies": {
+        "path-to-regexp": "1.7.0"
```

Comment:
💡 that makes much more sense. Updated, let me know if I should clean up the commit history for the branch.